AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/annotations/EBean.java

```diff
@@ -110,6 +110,22 @@
 		 */
 		Default, //
 
+		/**
+		 * A new instance of the bean is created the first time it is needed in an
+		 * activity scope, it is then retained and the same instance is always returned
+		 * from within the same activity. Different activities, hold different copies of
+		 * the bean
+		 */
+		Activity,
+
+		/**
+		 * A new instance of the bean is created the first time it is needed in aa
```

Comment:
A new instance of the bean is created the first time it is needed in a
Fragment scope, it is then retained and the same instance is always returned
from within the same Fragment. Different Fragments hold different copies of
the bean.